grammars/css.cson

```diff
@@ -20,294 +20,1218 @@
+              |
+              (["'][^"']+$)         # Unclosed quote
+              |
+              (?<=["'])             # After charset name
+              ([^;]+)               # Unexpected junk instead of semicolon
+            '''
+          }
+          {
+            'captures':
+              '1':
+                'name': 'keyword.control.at-rule.charset.css'
+              '2':
+                'name': 'punctuation.definition.keyword.css'
+            'match': '((@)charset)(?=\\s)'
+          }
+          {
+            'include': '#string-double'
+          }
+          {
+            'include': '#string-single'
+          }
+          {
+            'include': '#comment-block'
+          }
+        ]
+      }
+      {
+        # @import
+        'begin': '(?i)((@)import)(?:\\s+|$|(?=[\'"]|/\\*))'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.import.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': '\\s*(?=;|})'
+        'name': 'meta.at-rule.import.css'
+        'patterns': [
+          {
+            'begin': '\\G\\s*(?=/\\*)'
+            'end': '(?<=\\*/)\\s*'
+            'patterns': [
+              {
+                'include': '#comment-block'
+              }
+            ]
+          }
+          {
+            'include': '#string-double'
+          }
+          {
+            'include': '#string-single'
+          }
+          {
+            'include': '#comment-block'
+          }
+          {
+            'include': '#url'
+          }
+          {
+            'include': '#media-query-list'
+          }
+        ]
+      }
+      {
+        # @font-face
+        'begin': '(?i)((@)font-face)\\s*(?={|$|/\\*)'
+        'beginCaptures':
+          '1':
+            'name': 'keyword.control.at-rule.font-face.css'
+          '2':
+            'name': 'punctuation.definition.keyword.css'
+        'end': '(?!\\G)'
+        'name': 'meta.at-rule.font-face.css'
+        'patterns': [
+          {
+            'include': '#comment-block'
+          }
+          {
+            'include': '#escapes'
+          }
+          {
+            'include': '#rule-list'
+          }
+        ]
+      }
+      {
+        # @media
+        'begin': '(?i)(?=@media(?:\\s|\\(|$|/\\*))'
+        'end': '\\s*(})|(?=;)'
+        'endCaptures':
+          '1':
+            'name': 'punctuation.section.property-list.end.css'
+        'patterns': [
+          {
+            'begin': '(?i)(@)media(?=\\s|\\(|$|/\\*)'
+            'beginCaptures':
+              '0':
+                'name': 'keyword.control.at-rule.media.css'
+              '1':
+                'name': 'punctuation.definition.keyword.css'
+            'end': '\\s*(?=[;{])'
+            'name': 'meta.at-rule.media.css'
+            'patterns': [
+              {
+                'include': '#media-query-list'
+              }
+            ]
+          }
+          {
+            'begin': '\\s*({)'
```

Comment:
Why are we including whitespace in this capture?  Can't it just be `'begin': '{'`?